package.json

```diff
@@ -37,6 +37,7 @@
     "react-hook-form": "^7.54.0",
     "tailwind-merge": "^2.5.5",
     "tailwindcss-animate": "^1.0.7",
+    "use-debounce": "^10.0.4",
```

Comment:
wouldn't it better to create own use debounce hook rather than using npm package?

Reply:
wouldnt it be better to use react api - `useDeferredValue`

Reply:
In my opinion is much better to have npm package than making the same thing by ourselves, but I can change it if you want, doesn't matter